app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ApplicationMailer < ActionMailer::Base
+  default from: 'pre_deploy_checker@invoca.com'
```

Comment:
is this a working email group? otherwise what about just `development@`

Reply:
I kind of liked having it be separate, such that it can be easily filtered.